docs/slurm/main.md

```diff
@@ -38,15 +38,19 @@ There are two main ways to run programs on RCC clusters:
 
 * Passive via a "batch job" 
 
-In an interactive session, you will load [software modules](../software/faq.md) and run your scripts in real-time, whereas when submitting batch jobs, you specify the software modules to be loaded and scripts to be run in advance. 
+It is important to note that in both ways you are essentially submitting a job to the SLURM scheduler. You have to wait for the requested resource (i.e. number of CPU cores, memory, and GPUs, if any) to be allocated to your session or job.
```

Comment:
maybe, instead of "submitting a job to the SLURM scheduler." we say "submitting a job to the Slurm." or "submitting a job to the Slurm Workload Manager."

Reply:
fixed